echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiatorSpec.groovy

```diff
@@ -1,43 +1,33 @@
 package com.netflix.spinnaker.echo.pipelinetriggers.orca
 
-
+import com.netflix.spectator.api.DefaultRegistry
 import spock.lang.Specification
 import spock.lang.Subject
 import com.netflix.spinnaker.echo.model.Pipeline
-import org.springframework.boot.actuate.metrics.CounterService
+import spock.lang.Unroll
+
 import static rx.Observable.empty
 
 class PipelineInitiatorSpec extends Specification {
 
-  def counter = Stub(CounterService)
+  def registry = new DefaultRegistry()
```

Comment:
You can use `NoopRegistry` instead for tests.

Reply:
yeah I'll do that, I happened to chat with Brian H about that. I went with a DefaultRegistry because that's what the spectator testing docs show: https://github.com/Netflix/spectator/blob/master/docs/intro/testing.md